drivers/accel/adxl345/adxl345.c

```diff
@@ -40,7 +40,6 @@
 /******************************************************************************/
 /***************************** Include Files **********************************/
 /******************************************************************************/
-#include <stdint.h>
```

Comment:
i'm not entirely sure about this change as-is;
that is to say: it's not bad, but I don't know about the added value it adds;
nor the implications of doing it this way or another;

reasons for my un-sureness:
1. patch title is `drivers/accel: Make the header files self-sufficient` without providing details about the implications here; all it seems to do, is move stdint.h, which makes me assume it's for the int32_t/int8_t/etc types;
there isn't sufficient argumentation about why only stdint.h is moved and not other headers as well
2. since this code is not Linux, it gets copied/integrated all over, people will get various types/implementations of `stdint.h`; some header collisions could occur or missing definitions;
we may have to provide our own `stdint.h` for this code-base at some point to be a bit more comfortable with this point; but that's another point for another time

that being said: i am not against this as-is; nor am i convinced by this patch as-is

Reply:
1. I agree - the description was entirely missing.
2. I guess that this should be the subject of another discussion. The current patch just moves the include from .c to .h .